mojo/stdlib/stdlib/gpu/intrinsics.mojo

```diff
@@ -685,6 +704,33 @@ fn _get_nvtx_pointer_constraint() -> StaticString:
     return _get_nvtx_register_constraint[DType.int]()
 
 
+struct AirMemFlags:
```

Comment:
Since these types (AirMemFlags, AirScope, AirMemOrder) are only implementation details for the `load/store` functions - can we make these private (add and underscore `_` prefix to them)